library/cwm/src/lib/cwm/replace_point.rb

```diff
@@ -20,26 +20,18 @@ def initialize(id: "_placeholder", widget: Empty.new("_initial_placeholder"))
 
     # @return [UITerm]
     def contents
-      ReplacePoint(Id(widget_id), Empty(Id("____CWM_empty")))
-    end
-
-    # Switches to initial widgets
-    def init
```

Comment:
yeah, in the end I think it make sense to really use init as it was before. Reason is that if it is part of content, then CWM itself handle it. So Replace point should not do it, but when it is replaced, it will be handled by replace point. Which is not so easy to see. With old approach that init do replace, then from beginning all content events and callbacks are handled in replace point.